tbx/static_src/sass/main.scss

```
@@ -83,3 +83,6 @@
 @import 'components/values';
 @import 'components/utilities';
 @import 'components/webstory-player';
+
+// Others
```

Nitpick:

```suggestion
// CSS from external vendors
```

This is super minor - I was going to write it as `Vendors` to match all the other comments in this file, however that might get confused with browser vendors, which are more typically associated with CSS